src/view.js

```diff
@@ -0,0 +1,223 @@
+     this.drawPathBoard();
+   else if (this.game.board instanceof GraphBoard)
+     this.drawGraphBoard();
+   else 
+     do nothing and defer drawing to user? 
+
+   */
+};
+
+View.prototype.drawGridBoard = function() {
+  
+  this.boardView.x = c.boardStartX;
+  this.boardView.y = c.boardStartY;
+  this.boardView.beginFill(c.blueColor, 1);
+  this.boardView.drawRect(0, 0, c.boardWidth, c.boardHeight);
+  this.stage.addChild(this.boardView);
+  this.drawTiles(this.boardView);
+  this.drawTokens();
+  this.drawMessage();
+  this.animate();
+};
+
+// todo 
+View.prototype.drawPathBoard = function() {
+
+};
+
+// todo 
+View.prototype.drawGraphBoard = function() {
+
+};
+
+View.prototype.drawTile = function(opts) { 
+  var tileView = new PIXI.Graphics();
+  tileView.lineStyle(1, 0, 1);
+  tileView.beginFill(opts.tile.color, 1);
+  tileView.drawRect(0, 0, opts.dim.width, opts.dim.height);
+  tileView.x = opts.position.x;
+  tileView.y = opts.position.y;
+  
+  if (this.game.moveType == c.moveTypeManual) { 
+    tileView.interactive = true;
+    var context = this;
+    tileView.click = function(mouseData) {
+      /* make sure we're in the right state, 
+         a token has been pressed, 
+         and we're not a tile with a token on it (if we have > 0
+         children, then this click was meant for a token... */
+      if (context.turnMap.getCurrentState() == "waitingForMove" && 
+          context.game.proposedMove.token && 
+          tileView.children.length == 0) { 
+        context.game.setProposedMoveDestination(opts.tile);
+        context.turnMap.updateState("makeMove");
+      } 
+    };
+  } 
+  
+  this.tileViews[opts.index.x][opts.index.y] = tileView;
+  this.boardView.addChild(tileView);
+};
+
+View.prototype.drawTiles = function(boardView) {
+
+  var tileWidth = c.boardWidth / this.game.board.spaces[0].length;
+  var tileHeight = c.boardHeight / this.game.board.spaces.length;
+  var y_pos = c.boardHeight;
+  var x_pos = 0;
+  
+  for (var y = 0; y < this.game.board.spaces.length; y++) {
+    x_pos = 0;
+    y_pos -= tileHeight;
+    for (var x = 0; x < this.game.board.spaces[0].length; x++) {
+      var tile = this.game.board.getSpace(x, y);
+      this.drawTile({
+        tile: tile, 
+        boardView: boardView,
+        position: {x: x_pos, y: y_pos},
+        index: {x: x, y: y},
+        dim: {width: tileWidth, height: tileHeight}
+      });
+      x_pos += tileWidth;
+    }
+   }
+};
+
+// draws token, puts it on appropriate tile,
+// adds it to tokenViews
+View.prototype.drawToken = function(token) {
+
+  // draw the token 
+  var tokenView = new PIXI.Graphics();
+  tokenView.lineStyle(1, 0, 1);
+  tokenView.beginFill(token.color, 1);
+  if (this.game.moveType == c.moveTypeManual) { 
+    tokenView.interactive = true;
+    var context = this;
+    tokenView.click = function(mouseData) { 
+      if (context.turnMap.getCurrentState() == "waitingForMove") { 
+        var token = context.tokenViews.filter(function(tv) { 
```

Comment:
You have token as a parameter to the function and then you're redefining it here. I'm not sure which token you're trying to use down on line 131